awscli/examples/iam/list-server-certificates.rst

```diff
@@ -0,0 +1,32 @@
+**To list the server certificates in your AWS account**
+
+The following ``list-server-certificates`` command lists all of the server certificates stored and available for use in your AWS account. ::
+
+    aws iam list-server-certificates
+
+Output:
+
+    {
```

Comment:
Does this render correctly? The `rst` highlighter is not coloring it correctly here and the spacing is different than the other example files.

Reply:
I just fixed all of these.  As to the question about the last, yes, it renders correctly in my local build, which you can view at http://bisdavid.aka.amazon.com/docs/localcli/reference/iam/list-server-certificates.html    

I also added a few more (server-certificate CRUD commands) examples.

Please let me know if you find any other issues.